Pipfile

```diff
@@ -20,4 +21,4 @@ pytest = "*"
 python_version = "3.11"
 
 [scripts]
-sapinvoices = "python -c \"from sapinvoices.cli import main; main()\""
+sapinvoices = "python -c \"from sapinvoices.cli import cli; cli()\""
```

Comment:
You might consider changing this command to `sap` and the subcommands in the cli main group to something like `create-sandbox-data` and `process-invoices`. Running the full command right now as `pipenv run sapinvoices sap-invoices` feels a little odd.